policies/privacy-policy.md

```diff
@@ -0,0 +1,122 @@
+- **Committee and Member Display**: We publicly display committee members’ full names and GitHub logins (or alternate display names if specified) during their tenure.
+- **Communication**: For members using the ForwardEmail service, we facilitate communication through the provided forwarding addresses.
+- **Website Improvement**: Analytics data helps us improve our website’s usability and performance without identifying individual users.
+- **Project Management and Contribution Oversight**: We use contributor information to help projects monitor contribution activity, ensure adherence to contribution requirements (such as CLAs or DCOs), and assess overall project health.
+
+## Information Sharing and Disclosure
+
+We do not sell or rent your personal information. Information is shared under the following circumstances:
+
+- **Third-party services**: We do not proactively share your personal information with third-party service providers.
+    If you opt to use services like ForwardEmail, your information may be shared with those services as needed for their operation.
+    This sharing only occurs with your consent when you choose to use these services.
+- **Legal Requirements**: We may disclose information if required by law or in response to a valid legal request.
+
+## How We Protect Your Information
+
+We take reasonable measures necessary to protect your personal data from unauthorized access, alteration, or destruction; maintain data accuracy; and help ensure the appropriate use of your personal data.
+
+## Cross-border data transfers
+
+We may store and process your personal data in your region, in the United States, and in any other country where we or our service providers operate facilities.
+
+We may transfer personal data from the European Union, the United Kingdom, and Switzerland to other countries, some of which have not yet been determined by the European Commission to have an adequate level of data protection.
+For example, their laws may not guarantee you the same rights, or there may not be a privacy supervisory authority there that is capable of addressing your complaints.
+When we engage in such transfers, we use the Standard Contractual Clauses approved by the European Commission to help protect your rights and enable these protections to travel with your data.
+
+## Data Retention
+
+We retain information for as long as necessary to fulfill the purposes outlined in this policy, or as required by law.
+Session cookies are temporary and expire once your session ends.
+
+## Your Rights
+
+You have the right to request access to, correction, or deletion of your personal information.
+If you wish to exercise these rights, send an email to the [`legal` mailing list][CONTACTS.yaml].
+
+## Changes to This Policy
+
+We may update this Privacy Policy periodically to reflect changes in our practices or legal obligations.
```

Comment:
We may need to indicate how it's done also?
Nvm if not.

Reply:
```suggestion
We may update this Privacy Policy periodically to reflect changes in our practices or legal obligations using our [amendment process][].
```

Reply:
Yup, makes sense. Thanks!